dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -1262,6 +1338,9 @@ class PerfTest {
   /// Number of seconds to time out the test after, allowing debug callbacks to run.
   final int? timeoutSeconds;
 
+  /// Bool representing if this script is running in unit test mode.
```

Comment:
Here and elsewhere: Instead of having this opaque flag, would it make more sense for this and StartupTest to take a `String testOutputDirectory` parameter? That parameter would be a little more self-explanatory.

Reply:
Removed unittestMode variable from everywhere. I found an easier way to implement the fix using the default path everywhere and copy the logs to `FLUTTER_TEST_OUTPUTS_DIR` if set.